### PR TITLE
WIP: Simplify webui imports

### PR DESCRIPTION
**Pull request by VladimirSlavik**
WIP - would like to do as cockpit does with `parent.py`, but it's hardly one step

### CODE REVIEW

**Comment by VladimirSlavik** [comment]
I believe I'll just call the next commit "aaaargh" ;-)

**Comment by M4rtinK** [comment]
IMHO definitely going in the right direction! :)

**Comment by VladimirSlavik** [comment]
Just dumping the relevant parts for further investigation for myself...

<details>
<summary>Excerpt from IRC discussion on libera.chat's `#cockpit`</summary>

```
<vladimirslavik>  I'm trying to clean up imports in anaconda tests, and I'd like to start locally the cockpit container that creates the vms etc.
<vladimirslavik> what would be a good starting point?
<vladimirslavik> is the system the same as running the tests locally, or is there some difference?
<pitti> vladimirslavik: hey!
<vladimirslavik> it's related to this, btw. https://github.com/rhinstaller/anaconda/pull/4526
<pitti> vladimirslavik: sure, I actually do this all the time -- https://quay.io/repository/cockpit/tasks is set up to work directly with toolbox
<vladimirslavik> oooh, shiny! thanks!
<pitti> vladimirslavik: toolbox create --image quay.io/cockpit/tasks -c cockpit
<pitti> vladimirslavik: if you know toolbox already, that's really all you need to know; if not, happy to give summary and pointers
<vladimirslavik> do you prefer that to podman?
<pitti> vladimirslavik: it's not actually all that magic -- it's by and large just a F37 container with packages that we need, like firefox, chromium, libvirt, and qemu
<pitti> vladimirslavik: our bots run that container with podman for isolation; locally I prefer toolbox as I have my home dir etc. in the container, much easier to run stuff
<jkonecny[m]> distrobox is able to create it even as root when needed
<jkonecny[m]> can recommend distrobox - lot of great features
<vladimirslavik> ah ok, I'm not too used to toolbox, so I'll fight my way through that I guess :)
<pitti> in fact, cockpit/tasks *is* my environment -- all my terminals are inside that toolbox; I install a bunch of extra packages (like neovim, man-pages, etc.)
<pitti> vladimirslavik: you are welcome to use it with podman directly of course -- you can bind-mount your ~/.cache/cockpit* and your checkout with -v
<pitti> bu do look at toolbox or distrobox (they are very similar)
<jkonecny[m]> I have this: https://github.com/jkonecny12/toolbox-containers
<pitti> it's been a game changer for me
* Aussie_matt_ has quit (Quit: http://quassel-irc.org - Chat comfortably. Anywhere.)
<pitti> reproducible and non-previous dev environments, and the host can be very clean (no compilers, dev packages etc., read-only ostree)
<vladimirslavik> thanks, guess I will have to review those :)
<pitti> anyway, running o'clock! have a nice weekend everyone
<vladimirslavik> have a great weekend!
<vladimirslavik> and thanks again!
<pitti> https://github.com/cockpit-project/cockpituous/tree/main/tasks#using-with-toolbx
```
</details>

File: ui/webui/test/check-progress [hunk 1]
```diff
@@ -15,18 +15,13 @@
...
-sys.path.append(os.path.join(TEST_DIR, "common"))
-sys.path.append(os.path.join(os.path.dirname(TEST_DIR), "bots/machine"))
+from .parent import setup_paths
+setup_paths()
 
-from installer import Installer
-from testlib import MachineCase, test_main  # pylint: disable=import-error
-from machine_install import VirtInstallMachine
+from .helpers.installer import Installer
+from common.testlib import MachineCase, test_main  # pylint: disable=import-error
```
Comment (poncovka) [inline ui/webui/test/check-progress:23]: It would be great if all imports of our or Cockpit helper libraries were only relative, or absolute.

**Comment by VladimirSlavik** [comment]
I don't think I have the space to investigate this, and the issue will become self-evident later anyway.

**Pull request closed without merging**